servers/partner-ci/common.yaml

```diff
@@ -1,10 +1,14 @@
 - parameter:
     name: common-parameters
     parameters:
+      - string:
+          name: FUEL_RELEASE_VERSION
+          default: '9.0'
```

Comment:
I thought about it. But for that case we should create some sets of plugins jobs for each version of fuel I meant 9.0, 9.1, 9.2